Source/WebKit/Configurations/WebKit.xcconfig

```diff
@@ -211,11 +211,12 @@ OTHER_TAPI_FLAGS_SRCROOT_YES = -Xparser -include -Xparser config.h;
 SECTORDER_FLAGS = $(SECTORDER_FLAGS_$(CONFIGURATION));
 SECTORDER_FLAGS_Production[sdk=macosx*] = -Wl,-order_file,mac/WebKit2.order;
 
-EXCLUDED_SOURCE_FILE_NAMES = $(EXCLUDED_PRODUCT_DEPENDENCY_NAMES) $(EXCLUDED_IOS_RESOURCE_FILE_NAMES) $(EXCLUDED_MACOS_PLUGIN_FILE_NAMES) $(EXCLUDED_MIGRATED_HEADERS_COCOA_TOUCH_$(WK_IS_COCOA_TOUCH));
+EXCLUDED_SOURCE_FILE_NAMES = $(EXCLUDED_PRODUCT_DEPENDENCY_NAMES) $(EXCLUDED_IOS_RESOURCE_FILE_NAMES) $(EXCLUDED_MACOS_SOURCE_FILE_NAMES) $(EXCLUDED_MIGRATED_HEADERS_COCOA_TOUCH_$(WK_IS_COCOA_TOUCH));
 // These are only available in internal builds, so they must be explicitly ignored for the open-source build to succeed.
 EXCLUDED_PRODUCT_DEPENDENCY_NAMES = libWebKitAdditions.a AuthenticationServicesCore.framework;
 EXCLUDED_IOS_RESOURCE_FILE_NAMES = Resources/ios/*;
 EXCLUDED_IOS_RESOURCE_FILE_NAMES[sdk=iphone*] = ;
+EXCLUDED_MACOS_SOURCE_FILE_NAMES[sdk=iphone*] = Resources/mac/* com.apple.WebKit.NetworkProcess.sb.in com.apple.WebKit.GPUProcess.sb.in com.apple.WebKit.webpushd.mac.sb.in com.apple.WebProcess.sb.in;
```

Comment:
Looks like `Resources/mac` is gone too, but I'm less certain about losing `*.pdf` from here.

Reply:
Good catch. I just checked, and there are no *.pdf files tracked in WebKit.xcodeproj. The exclude rule dates back to 2013, and unfortunately doesn't have a much context: d41656c0f04e00b3f3a48c0887282af31a442f0e

There are some PDFs inside of `iOS.xcassets`, but they are not known to the build system (since it processes the asset catalog as a unit).